source/fe/fe_nedelec_sz.cc

```diff
@@ -45,6 +46,104 @@ FE_NedelecSZ<dim, spacedim>::FE_NedelecSZ(const unsigned int order)
 
   // Generate the 1-D polynomial basis.
   create_polynomials(order);
+
+  // Compute the face embedding.
+  FullMatrix<double> face_embeddings[GeometryInfo<dim>::max_children_per_face];
+
+  // TODO: The implementation assumes that all faces have the same
+  //       number of DoFs.
```

Comment:
Why did you set a `TODO` here? I think there are several more things to be done if we really wanted to use the NedelecSZ elements on something like a wedge or a pyramid, so I feel that we should say
```suggestion
  // The implementation assumes that all faces have the same
  // number of DoFs.
```

Reply:
I am not sure anymore why I placed that `TODO`. I guess I was not sure if I could keep using that assumption.

I removed that TODO.